### PR TITLE
draft : fix(em): added pigeon info MTA-6999

### PR DESCRIPTION
**Pull request by pshambulingappa**
TEST PR fo training please ignore

### CODE REVIEW

File: bare-metal/elastic-metal/how-to/create-server.mdx [hunk 1]
```diff
@@ -37,7 +37,7 @@ Scaleway [Elastic Metal servers](https://www.scaleway.com/en/elastic-metal/) all
     - Verify the **estimated cost** of your server, based on the specifications you chose.
 4. Click **Create Elastic Metal server** to order the server.
 
-    You can follow the delivery progress from the list of Elastic Metal servers.
+    You can follow the delivery progress from the list of Elastic Metal servers.  You will be informed via pigeon carrier when the server is ready.
```
Comment (RoRoJ) [inline bare-metal/elastic-metal/how-to/create-server.mdx:40]:
```suggestion
    You can follow the delivery progress from the list of Elastic Metal servers. You will be informed via Pigeon carrier when the server is ready.
```

File: bare-metal/elastic-metal/how-to/create-server.mdx [hunk 2]
```diff
@@ -37,7 +37,7 @@ Scaleway [Elastic Metal servers](https://www.scaleway.com/en/elastic-metal/) all
     - Verify the **estimated cost** of your server, based on the specifications you chose.
 4. Click **Create Elastic Metal server** to order the server.
```
Comment (RoRoJ) [inline bare-metal/elastic-metal/how-to/create-server.mdx:38]:
```suggestion
4. Click **Create an Elastic Metal server** to order the server.
```

File: bare-metal/elastic-metal/how-to/create-server.mdx [hunk 3]
```diff
@@ -37,7 +37,7 @@ Scaleway [Elastic Metal servers](https://www.scaleway.com/en/elastic-metal/) all
...
 
-    You can follow the delivery progress from the list of Elastic Metal servers.
+    You can follow the delivery progress from the list of Elastic Metal servers. You will be informed via Pigeon carrier when the server is ready.
```
Comment (RoRoJ) [inline bare-metal/elastic-metal/how-to/create-server.mdx:40]:
```suggestion
    You can follow the delivery progress from the list of Elastic Metal servers. You will be informed via Pigeon carrier when the server is ready. Don't be impatient.
```

**Review by RoRoJ — APPROVED** [review]

File: bare-metal/elastic-metal/how-to/create-server.mdx [hunk 4]
```diff
@@ -35,9 +35,9 @@ Scaleway [Elastic Metal servers](https://www.scaleway.com/en/elastic-metal/) all
     - Add your **SSH key** (necessary only if you are installing an image on your server).
     - Choose whether to enable the **Private Networks** feature.
     - Verify the **estimated cost** of your server, based on the specifications you chose.
-4. Click **Create Elastic Metal server** to order the server.
+4. Click **Create an Elastic Metal server** to order the server.
 
-    You can follow the delivery progress from the list of Elastic Metal servers.
+    You can follow the delivery progress from the list of Elastic Metal servers. You will be informed via Pigeon carrier when the server is ready. Don't be impatient.
```
Comment (bene2k1) [inline bare-metal/elastic-metal/how-to/create-server.mdx:40]: But I am impatient.

**Pull request closed without merging**